scripts/test_philips_tiff.py

```diff
@@ -0,0 +1,339 @@
+#!/usr/bin/env python3
+# SPDX-FileCopyrightText: Copyright (c) 2025, NVIDIA CORPORATION. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+"""Test Philips TIFF support in cuslide2"""
+
+import json
+import os
+import sys
+import time
+from pathlib import Path
+
+
+def setup_environment():
+    """Setup cuCIM environment for cuslide2 plugin"""
+
+    # Get current build directory
+    repo_root = Path(__file__).parent.parent
+    plugin_lib = repo_root / "cpp/plugins/cucim.kit.cuslide2/build-release/lib"
+
+    if not plugin_lib.exists():
+        plugin_lib = repo_root / "install/lib"
+
+    # Read version from VERSION file
+    version_file = repo_root / "VERSION"
+    if version_file.exists():
+        version = version_file.read_text().strip()
+    else:
+        version = "25.12.00"  # Fallback version
```

Comment:
Same question about version handling here. Though we can discuss in the previous thread. Just marking this instance so we can track